client/me/account/main.jsx

```diff
@@ -393,18 +406,18 @@ const Account = createReactClass( {
 	},
 
 	handleSubmitButtonClick() {
-		const { unsavedSettings } = this.props.userSettings;
+		const { unsavedUserSettings } = this.props;
 		this.recordClickEvent( 'Save Account Settings Button' );
-		if ( has( unsavedSettings, colorSchemeKey ) ) {
-			const colorScheme = get( unsavedSettings, colorSchemeKey );
+		if ( has( unsavedUserSettings, colorSchemeKey ) ) {
```

Comment:
Should we use `this.hasUnsavedUserSetting( colorSchemeKey )` here?

Reply:
Yes, we can do that 👍🏻

Reply:
See https://github.com/Automattic/wp-calypso/pull/49760/commits/1cc66281e1f35db279a3d90db5eb6e3ad70678bd